### PR TITLE
Dev docs

### PR DESCRIPTION
**Pull request by sekka1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,3 +97,27 @@ kubernetes-state-checker:
...
+```
+go get gopkg.in/yaml.v2
+go get k8s.io/apimachinery/pkg/api/errors
+go get k8s.io/apimachinery/pkg/apis/meta/v1
+go get k8s.io/client-go/kubernetes
+go get k8s.io/client-go/tools/clientcmd
+go get k8s.io/client-go/util/homedir
+```
+
+Add in a kubeconfig that you want to use:
```
Comment (mazamats) [inline README.md:119]: Any way we can auto volume mount these from the users `~/.kube`?

File: README.md [hunk 2]
```diff
@@ -97,3 +97,27 @@ kubernetes-state-checker:
...
+Once you install this, and restart VScode in this repository, it will ask if you want to open it in a container.  The first time you do this, it will take some time since it is building the container based on the Dockerfile in the `.devcontainer` directory.  After it builds the container, it will open this project inside of this container and you can code away as normal.
+
+Install the go dependencies:
+```
+go get gopkg.in/yaml.v2
+go get k8s.io/apimachinery/pkg/api/errors
+go get k8s.io/apimachinery/pkg/apis/meta/v1
+go get k8s.io/client-go/kubernetes
+go get k8s.io/client-go/tools/clientcmd
+go get k8s.io/client-go/util/homedir
```
Comment (mazamats) [inline README.md:116]: Can we put these in a `go.mod` file or something so its one command to get all of it?

File: README.md (hunk 1 above)
Comment (mazamats) [reply README.md:119]: https://code.visualstudio.com/docs/remote/containers-advanced#_adding-another-local-file-mount

File: README.md (hunk 2 above)
Comment (sekka1) [reply README.md:116]: I was trying to do that initially but it was breaking all of the auto complete and libs location in the vscode container.  Can we push this off to another ticket?

**Review by mazamats — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (sekka1) [reply README.md]: Good idea...i updated this.

File: README.md (hunk 2 above)
Comment (mazamats) [reply README.md:116]: Sounds good to me

Comment (bpeskin-anthemai) [reply README.md:116]: +1 on an improvement ticket

Comment (sekka1) [reply README.md:116]: Ticket: https://anthemai.atlassian.net/browse/HSS-218

**Pull request merged**